doc/setup/quickstart_cmake/index.md

```diff
@@ -1,3 +1,236 @@
 # Quick Setup (using CMake) {#setup}
 
-Please see the SeqAn documentation about [setup](https://docs.seqan.de/seqan/3-master-user/setup.html).
+<b>Learning Objective:</b><br>
+In this short guide you will learn how to set up Sharg and how to compile a small example to test whether everything
+works.
+
+\tutorial_head{Easy, 30 Minutes, No prerequisites, }
+
+[TOC]
+
+<br>
+
+# Software
+Requirements:
+  - gcc >= 10
+  - cmake >= 3.4
+  - git
+
+## Installing GCC
+
+SeqAn requires GCC >= 10. Current versions of LLVM/Clang and VisualStudio/MSVC are **not yet supported**.
+We will briefly explain how to install GCC-11 (or the latest GCC if such an option is available) on some popular
+operating systems. We recommend using the latest version of GCC available. For more information, refer to your
+operating system's documentation.
+
+\startcollapsible{Linux-based}
+
+#### Ubuntu >= 22.04
+```bash
+# Installs default compiler version (gcc-11 for Ubuntu 22.04).
+sudo apt install g++
+# To install gcc-12, follow below instructions.
+```
+
+#### Ubuntu < 22.04
+```bash
+sudo add-apt-repository --no-update --yes ppa:ubuntu-toolchain-r/ppa
+sudo add-apt-repository --no-update --yes ppa:ubuntu-toolchain-r/test
+sudo apt-get update
+sudo apt install g++-11
+```
+
+#### Using [conda](https://conda.io)
+To avoid interference with system packages, we recommend creating a new environment when using conda.
+```bash
+conda create -n conda_gcc_env -c conda-forge gcc_linux-64
+conda activate conda_gcc_env
+```
+This will put GCC in a separate environment `conda_gcc_env` which can be activated via `conda activate conda_gcc_env`
+and deactivated via `conda deactivate`.
+
+\endcollapsible
+
+\startcollapsible{macOS}
+
+#### Using [Homebrew](https://brew.sh/)
+```bash
+brew install gcc@11
+```
+
+#### Using [MacPorts](https://www.macports.org/)
+```bash
+sudo port install gcc11
+```
+
+\endcollapsible
+
+\startcollapsible{Windows}
+
+#### Using [WSL](https://docs.microsoft.com/en-us/windows/wsl/about)
+The Windows Subsystem for Linux offers an easy way to run a Linux distribution under Windows.
+Follow [Microsoft's setup guide](https://docs.microsoft.com/en-us/windows/wsl/about) to install WSL and then follow
+the steps listed for Linux-based systems.
+
+\endcollapsible
+
+\startcollapsible{Browser-based}
+
+#### Using [gitpod.io](https://gitpod.io/#https://github.com/seqan/sharg/)
+[gitpod.io](https://gitpod.io) allows you to edit, compile and run code from within your browser. The free version includes 50
+hours of use per month, which is plenty for our tutorials. A GitHub account is required.
+[Click here](https://gitpod.io/#https://github.com/seqan/sharg/) to open sharg in gitpod.
```

Comment:
```suggestion
[Click here](https://gitpod.io/#https://github.com/seqan/sharg-parser/) to open sharg in gitpod.
```

Reply:
Thought for future adjustments:
We should somehow link to a different URL. Just having sharg-parser (I guess this is the same for seqan3) opened in gitpod.io is not so useful. It doesn't allow to follow the tutorial below.

Reply:
hm good point. We should remove the gitpod link then for now.